spec/index.html

```diff
@@ -495,6 +496,25 @@ <h2>RDF Documents and Syntaxes</h2>
     and still the subject of group discussion. An alternative to conformance
     levels, "profiles", may be adopted instead, abandoned, or described in 
     another specification.</p>
+
+  <section id="rdf-strings">
+    <h2>Strings in RDF</h2>
+
+    <p>RDF uses Unicode [[Unicode]] as the fundamental representation for string values.
+      Within this, and related specifications, the term <dfn id="dfn-rdf-string">string</dfn>
+      is used to describe an ordered sequence of zero or more <a data-lt="code point" class="lint-ignore">Unicode code points</a> which are <a data-cite="i18n-glossary#dfn-scalar-value" class="lint-ignore">Unicode scalar values</a>, which do not include Unicode surrogate code points.
+      in any <a data-cite="i18n-glossary#dfn-character-encoding">character encoding</a>.
```

Comment:
```suggestion
      is used to describe an ordered sequence of zero or more <a data-lt="code point" class="lint-ignore">Unicode code points</a> which are <a data-cite="i18n-glossary#dfn-scalar-value" class="lint-ignore">Unicode scalar values</a>. Unicode scalar values do not include the high-surrogate or low-surrogate code points.
```
This fixes some sentence damage in the working text - "code points. in any"

The language in the Unicode spec seems to be e.g. "high-surrogate and low-surrogate code points are designated for surrogate code units"

The "in any character encoding" is confusing given UTF-16. The key point about UTF-8 is covered by the next sentence. Encoding for XML is format sis left to XML (it doesn't even have to be unicode).

Reply:
I've found places in the Uniocde spec that say "surrogate code points" (D15). While most places say "high-surrogate ... low-surrogate", 

Suggestion chnaged:
--
do not
include the high-surrogate or low-surrogate code points.
=>
do not
include the surrogate code points.
--